packages/create-farm/templates/react/package.json

```diff
@@ -7,8 +7,8 @@
     "react-dom": "18"
   },
   "devDependencies": {
-    "@farmfe/cli": "^0.5.1",
-    "@farmfe/core": "^0.6.4",
+    "@farmfe/cli": "^0.5.2",
+    "@farmfe/core": "^0.7.6",
     "@farmfe/plugin-react": "0.1.3",
```

Comment:
the latest version of plugin-react is 0.1.5

Reply:
It's done.